### ISSUE TITLE
Broken unit test lib/apiservers/engine/backends/image_test

### ISSUE THREAD

**Issue by hickeng** [vmware/vic#1158]
https://github.com/vmware/vic/blame/master/lib/apiservers/engine/backends/image_test.go#L33
```
++ echo github.com/vmware/vic/lib/apiservers/engine/backends
+ f=/C/Users/George/Documents/VMware/VIC/src/github.com/vmware/vic/.cover/github.com-vmware-vic-lib-apiservers-engine-backends.cover
+ go test -v -covermode=count -coverprofile=/C/Users/George/Documents/VMware/VIC/src/github.com/vmware/vic/.cover/github.com-vmware-vic-lib-apiservers-engine-backends.cover github.com/vmware/vic/lib/apiservers/engine/backends
# github.com/vmware/vic/lib/apiservers/engine/backends
/C/Users/George/Documents/VMware/VIC/src/github.com/vmware/vic/lib/apiservers/engine/backends/image_test.go:33: undefined: getV1MapFromImages
/C/Users/George/Documents/VMware/VIC/src/github.com/vmware/vic/lib/apiservers/engine/backends/image_test.go:51: undefined: getV1MapFromImages
/C/Users/George/Documents/VMware/VIC/src/github.com/vmware/vic/lib/apiservers/engine/backends/image_test.go:89: undefined: getV1MapFromImages
/C/Users/George/Documents/VMware/VIC/src/github.com/vmware/vic/lib/apiservers/engine/backends/image_test.go:100: undefined: resolveImageSizes
/C/Users/George/Documents/VMware/VIC/src/github.com/vmware/vic/lib/apiservers/engine/backends/image_test.go:118: cannot use v1Image (type *image.V1Image) as type *metadata.ImageConfig in argument to convertV1ImageToDockerImage
FAIL    github.com/vmware/vic/lib/apiservers/engine/backends [build failed]
Makefile:195: recipe for target 'test' failed
```

The fact this isn't being run by the make test target is what I'm currently looking into.

**Comment by hickeng** [comment]
@jzt can I simply delete the image_test.go file to fix this?

**Comment by jzt** [comment]
Fixed in #1187.
